torchbenchmark/models/tts_angular/angular_tts_main.py

```diff
@@ -0,0 +1,324 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import argparse
+import os
+import sys
+import time
+import traceback
+import math
+
+import torch
+import torch as T
+from .model import SpeakerEncoder, AngleProtoLoss
+from .audio import AudioProcessor
```

Comment:
We are using generated input instead of actual audio file, so all the `audio.py` can be removed, I guess?